Libraries/LibJS/Parser.cpp

```diff
@@ -1089,17 +1113,29 @@ NonnullRefPtr<ThrowStatement> Parser::parse_throw_statement()
 NonnullRefPtr<BreakStatement> Parser::parse_break_statement()
 {
     consume(TokenType::Break);
+    FlyString target_label;
+    if (match(TokenType::Semicolon)) {
+        consume();
+        return create_ast_node<BreakStatement>(target_label);
+    }
+    if (match_identifier_name() && !m_parser_state.m_current_token.trivia().contains('\n'))
```

Comment:
`match(TokenType::Identifier)` instead of `match_identifier_name()`. Labels can not be reserved keywords.